pkg/doc/bbs/bbs12381g2pub/internal/kilic/bls12-381/g1_custom.go

```diff
@@ -0,0 +1,520 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bls12381
+
+import (
+	"golang.org/x/crypto/blake2b"
+	"hash"
+)
+
+func HashToCurve(msg, domain []byte) ([]byte, error) {
```

Comment:
Should avoid hardcoding to blake2b here :).

Reply:
Well, `SHA-256` is hardcoded in `kilic`, and [BBS+ signature schema](https://mattrglobal.github.io/bbs-signatures-spec/) strictly defines that `blake2b` hash function should be used, that's why I decided to hardcode this.
@troyronda what do you think?

Reply:
@kdimak I think we shouldn't hardcode it since we are trying to raise a PR that states that :).

Reply:
OK, I will change this.